prisma/schema.prisma

```diff
@@ -161,8 +161,9 @@ enum UserCreationRequestStatus {
 }
 
 model UserCreationRequest {
-  id   String  @id @default(nanoid())
-  type String?
+  id            String  @id @default(nanoid())
+  type          String?
+  existedUserId String?
```

Comment:
maybe existingUserId? and maybe it worth to link existing user to decree request